week-10-angular/day-02/module-01/readme.md

```diff
@@ -1 +1,223 @@
 # <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Intro Angular
+
+| **Learning Objectives** |
+| :---- |
+| Grok Angular's philosophy |
+| Initialize Angular in an HTML view |
+| Experiment with templating & expressions |
+| Organize your code with controllers |
+| Pass data between the View & Controller with $scope|
+| Leverage built-in filters |
```

Comment:
Are you going to be going over filters and directives? My lesson notes for the afternoon heavily cover those.